webv2/typemap/postgres_typemap.go

```diff
@@ -152,6 +152,13 @@ func ToSpannerTypePostgres(srcType string, spType string, mods []int64) (ddl.Typ
 		default:
 			return ddl.Type{Name: ddl.Timestamp}, []internal.SchemaIssue{internal.Timestamp}
 		}
+	case "json":
```

Comment:
add jsonb as well as in postgres/toddl.go

Reply:
Updated